Documentation/development-environment.md

```diff
@@ -120,6 +120,49 @@ k8s-08    Ready     node          2m        v1.9.0+coreos.0
 k8s-09    Ready     node          2m        v1.9.0+coreos.0
 ```
 
+## Running the Kubernetes Dashboard UI
+
+kubespray sets up the Dashboard pod by default, but you must authenticate with a bearer token, even for localhost access with kubectl proxy.  To allow access, one possible solution is to:
+
+1) create an admin user by creating admin-user.yaml with these contents (and using kubectl -f create admin-user.yaml):
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: admin-user
+  namespace: kube-system
+```
+
+2) grant that user the ClusterRole authorization by creating and applying admin-user-cluster.role.yaml:
```

Comment:
did you mean `Grant the user...`?

Reply:
not sure what you mean here; can fix capitalization, but are you suggesting "the" instead of "that"?  I meant to specifically call out the user added in step 1.  Or maybe I'm missing something meta?

Reply:
Oh, I see I misread it. Yes, I guess now it's just a capitalization suggestion.